<?xml version="1.0" encoding="utf-8"?>

<launch>
	<!--<node name="kinect" pkg="openni" type="usb_cam_node" ></node> -->
	<!--<node pkg="image_view" type="image_view" name="image_view" output="screen">
		<remap from="image" to="camera/rgb/image_rect_color" />
		<param name="autosize" type="bool" value="true" />
	</node>-->

	<node pkg="ar_sys" type="single_board_occlusion" name="ar_single_board" output="screen">
		<remap from="/camera_info" to="/camera/rgb/camera_info" />
		<remap from="/image" to="/camera/rgb/image_rect_color" />
		
		<param name="marker_size" type="double" value="0.05"/>
		<!--<param name="board_config" type="string" value="$(find ar_sys)/data/single/board.yml"/>-->
		<!--<param name="board_config" type="string" value="/home/radek/Downloads/aruco-testdata/chessboard/chessboardinfo2_pix.yml"/>-->
		<!-- <param name="board_config" type="string" value="/home/radek/catkin_ws/src/kinect_io/data/chessboardinfo2_meters.yml"/> -->
		

		<param name="board_config" type="string" value="/home/radek/Downloads/aruco-1.2.5/build/utils_hrm/newboard.yml" />
		<param name="dictionary" type="string" value="/home/radek/Downloads/aruco-1.2.5/build/utils_hrm/test_dict.yml" />

		<param name="board_frame" type="string" value="/board1" />
		<param name="draw_markers" type="bool" value="true" />
		<param name="draw_markers_cube" type="bool" value="true" />
		<param name="draw_markers_axis" type="bool" value="false" />
	</node>

	<node pkg="image_view" type="image_view" name="image_view_rest" output="screen">
		<remap from="image" to="/ar_single_board/result" />
		<param name="autosize" type="bool" value="true" />
	</node>

	<!--<node ns="/camera/rgb" pkg="image_proc" type="image_proc" name="image_procg" />-->
	<!--<node ns="camera1" pkg="image_proc" type="image_proc" name="image_procg" />
	<node ns="camera1" pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen">
		<param name="width" type="int" value="1280" />
		<param name="height" type="int" value="1024" />
		<param name="fps" type="int" value="30" />
		<param name="frame_id" type="string" value="/camera1" />
		<param name="device" type="string" value="/dev/video0" />
		<param name="camera_info_url" type="string" value="file://$(find ar_sys)/camera_info/camera_calibration.yaml" />
	</node>

	<node pkg="ar_sys" type="single_board" name="ar_single_board" output="screen">
		<remap from="/camera/rgb/camera_info" to="/camera1/camera_info" />
		<remap from="/camera/rgb/image_raw" to="/camera1/image_rect_color" />
		
		<param name="frame_id" type="string" value="/camera/rgb/" />
		<param name="video_namespace" type="string" value="/camera/rgb/"/>
		<param name="camera_enable" type="bool" value="false"/>
		<param name="camera_device" type="string" value="false"/>
		<param name="video_rectified" type="string" value="true"/>

		<param name="video_image_topic" type="string" value="image_raw"/>
		<param name="video_info_topic" type="string" value="camera_info"/>
		

		<param name="image_is_rectified" type="bool" value="true"/>
		<param name="marker_size" type="double" value="0.05"/>
		<param name="board_config" type="string" value="$(find ar_sys)/data/single/board.yml"/>
		<param name="board_frame" type="string" value="/board1" />
		<param name="draw_markers" type="bool" value="false" />
		<param name="draw_markers_cube" type="bool" value="true" />
		<param name="draw_markers_axis" type="bool" value="false" />
	</node>

	<node pkg="image_view" type="image_view" name="image_view" output="screen">
		<remap from="image" to="/ar_single_board/result" />

		<param name="autosize" type="bool" value="true" />
	</node> -->

</launch>